projects/arlas-components/src/lib/components/mapgl/model/ArlasMapboxGL.ts

```diff
@@ -450,7 +449,7 @@ export class ArlasMapGL extends AbstractArlasMapGL implements MapOverride {
                                 collection?: string): void {
```

Comment:
This method could be moved to the abstract map, since we only use methods that are reimplemented by each map provider

Reply:
fixed